cypress/integration/host/host_events.ts

```diff
@@ -12,6 +12,12 @@ describe("Host events", () => {
 
   it("host events display the correct text", () => {
     cy.visit(pathWithEvents);
+    cy.dataTestId("host-event-table-page-size-selector").click();
+    cy.dataTestId("host-event-table-page-size-selector-100").click();
+
+    const dataCyTableRows =
+      "[data-test-id=host-events-table] tr td:first-child";
+    cy.get(dataCyTableRows).should("have.length.of.at.most", 100);
```

Comment:
i think this will always pass because it checks if task queue table length is less than or equal to 100. since you're working with static data, you can assert the exact length that the table should be

Reply:
good point. changed it